service/src/main/java/com/epam/esm/service/config/ServiceConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.service.config;
+
+import com.epam.esm.dao.config.DBConfig;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.sql.DataSource;
+
+@Configuration
+@Import({DBConfig.class})

```

Comment:
Could you explain what is the purpose of this setting?

Reply:
there's no need in using this annotation. removed